src/components/components/ebay-notice-base/index.marko

```diff
@@ -25,7 +28,7 @@ $ var prefixClass = input.prefixClass;
 <${input.root || 'section'}
     aria-labelledby=!input.noA11yLabel && component.elId('status')
     aria-roledescription=input.a11yRoleDescription
-    class=[prefixClass, input.class]
+    class=[prefixClass, input.class, input.status === 'education' && input.prominent ? `${prefixClass}--prominent` : '' ]
```

Comment:
Marko automatically removes `null | undefined | false` values from the class list so you can simplify this a little bit:
```suggestion
    class=[prefixClass, input.class, input.status === 'education' && input.prominent && `${prefixClass}--prominent`]
```

Reply:
Also, now that I am looking at it again, do you _need_ to also check that `status === 'education'`? I know in `skin` we say it is "strictly for education notices only" but I don't think we need to enforce that in JS.

Reply:
I guess we might need to as team tend to misuse all these attributes. Lets see others thoughts.

Reply:
I would say we should keep `status === 'education'` in the conditional as it removes DOM use cases that have no matching styles and reduces the likelihood of someone falsely expecting a "prominent" variation for other types of notices.